src/storage/src/client/upload_object_buffered.rs

```diff
@@ -806,32 +809,34 @@ where
     let mut process_buffer = |mut b: bytes::Bytes| match b.len() {
         n if size + n > target_size => {
             let remainder = b.split_off(target_size - size);
+            size += b.len();
```

Comment:
consider: `size = target_size;`

might be clearer?

Reply:
Done.